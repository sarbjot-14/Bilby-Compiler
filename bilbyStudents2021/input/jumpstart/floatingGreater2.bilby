main{
	print 124 > 66 $n;
}
