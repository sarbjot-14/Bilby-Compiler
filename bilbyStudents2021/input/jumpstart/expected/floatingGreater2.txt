Couldn't read file "/home/sarbjot/Desktop/cmpt379/bilbyCompiler/bilbyStudents2021/input/jumpstart/output/floatingGreater2.asm"
